yogstation/code/game/gamemodes/shadowling/shadowling_abilities.dm

```diff
@@ -112,7 +112,7 @@
 	var/admin_override = FALSE //Requested by Shadowlight213. Allows anyone to cast the spell, not just shadowlings.
 
 /obj/effect/proc_holder/spell/aoe_turf/proc/extinguishItem(obj/item/I, cold = FALSE) //Does not darken items held by mobs due to mobs having separate luminosity, use extinguishMob() or write your own proc.
-	var/blacklisted_lights = list(/obj/item/flashlight/flare, /obj/item/flashlight/slime)
+	var/blacklisted_lights = list(/obj/item/flashlight/flare, /obj/item/flashlight/slime, /obj/item/electronic_assembly)
```

Comment:
This does nothing, the circuitry assemblies aren't of type flashlight so it skips the if, therefore it completely skips the blacklisted_lights list. It seems the I.set_light(0) is there for precaution against such things, and it indeed should work against assemblies, however to evade that, it's as simple as adding a ticker, or 2 alternating lights, so you can still constantly have lights on. Basically this commit does nothing.